Data/Aeson/Text.hs

```diff
@@ -46,8 +46,9 @@ encodeToLazyText = LT.decodeUtf8 . encodingToLazyByteString . toEncoding
 -- embedded efficiently in a text-based protocol.
 --
 -- If you are going to immediately encode straight to a
--- 'L.ByteString', it is more efficient to use 'encodeToBuilder'
--- instead.
+-- 'L.ByteString', it is more efficient to use
+-- @'encodingToLazyByteString' . 'toEncoding'@ (lazy ByteString) or
```

Comment:
that's is just `encode` from `Data.Aeson`.

Reply:
eh.. right